agent/functional_tests/testdata/taskdefinitions/init-process/task-definition.json

```diff
@@ -0,0 +1,13 @@
+{
+  "family": "ecsftest-init-process",
+  "containerDefinitions": [{
+    "image": "127.0.0.1:51670/ubuntu:latest",
+    "name": "exit",
+    "cpu": 10,
+    "memory": 10,
+    "linuxParameters": {
+      "initProcessEnabled":true
+    },
+    "command": ["sh", "-c", "ps -x | grep \"/dev/init\" && exit 42 || exit 1"]
```

Comment:
This will return false positives I think. Can we do `pidof "/dev/init" == 1`  check instead?

Reply:
`pidof "/dev/init" == 1` is a better check. will change this.